layouts/partials/page-header.html

```diff
@@ -10,26 +10,29 @@
     {{ $parent := .page.Parent }}
     {{ if $parent }}
         {{ $name := (partial "title" $parent) }}
-        {{ $value := (printf "<a href='%s'>%s</a> / %s" $parent.RelPermalink $name .value) }}
-        {{ template "breadcrumb" dict "page" $parent "value" $value }}
+        {{ $position := (sub .position 1) }}
+        {{ $value := (printf "<li itemprop='itemListElement' itemscope itemtype='https://schema.org/ListItem'><a itemscope itemtype='https://schema.org/WebPage' itemprop='item' itemid='%s' href='%s'><span itemprop='name'>%s</span></a><meta itemprop='position' content='%d' /></li><li> / </li>%s" $parent.RelPermalink $parent.RelPermalink $name $position .value) }}
+        {{ template "breadcrumb" dict "page" $parent "value" $value "position" $position }}
     {{ else }}
         {{ .value | safeHTML }}
     {{ end }}
 {{ end }}
 
 {{ $showBreadcrumb := (and (default true .Page.Params.GeekdocBreadcrumb) (default true .Site.Params.GeekdocBreadcrumb)) }}
 {{ $showEdit := (and ($.Scratch.Get "geekdocFilePath") $geekdocRepo $geekdocEditPath) }}
-<div class="gdoc-page__header flex flex-wrap justify-between{{ if not $showEdit }} hidden-mobile{{ end }}{{ if (and (not $showBreadcrumb) (not $showEdit)) }} hidden {{ end }}" itemscope itemtype="http://data-vocabulary.org/Breadcrumb">
-    <span>
+<div class="gdoc-page__header flex flex-wrap justify-between{{ if not $showEdit }} hidden-mobile{{ end }}{{ if (and (not $showBreadcrumb) (not $showEdit)) }} hidden {{ end }}" itemprop="breadcrumb">
+    <div>
```

Comment:
@xoxys  Maybe move `div` under `{{ if $showBreadcrumb }}` to prevent empty `div` block in DOM when `$showBreadcrumb == false`?

Reply:
I've moved it back to prevent content shift if breadcrumbs are disabled. The empty div is required for the flex box layout to keep the "Edit page" link on the right side.

Reply:
and for mobile device change `justify-between` to `justify-content: flex-end`?

Reply:
It's not just the mobile view, but should work now without the empty divs for both showBreadcrumb and showEdit.

Reply:
@xoxys thanks! 👍